### PR TITLE
Search Results by Quiz Name

### PR DESCRIPTION
**Pull request by andreiuta97**
Search Results by Quiz Name
Change "Quiz Id" column into "Quiz Name"
PHPTRAININ-391: https://jira.evozon.com/browse/PHPTRAININ-391
PHPTRAININ-547: https://jira.evozon.com/browse/PHPTRAININ-547

Note: Branch created from search-quiz-by-description.

### CODE REVIEW

File: src/Entity/QuizInstance.php [hunk 1]
```diff
@@ -96,11 +96,11 @@ public function setName(string $name)
     }
 
     /**
-     * @param string $type
+     * @param string $description
      */
-    public function setType(string $type)
+    public function setDescription(string $description)
```
Comment (Raresmldvn) [inline src/Entity/QuizInstance.php]: return type `void`. even if you don't return anything, we must indicate the return type

File: src/Entity/QuizTemplate.php [hunk 2]
```diff
@@ -70,11 +70,11 @@ public function setName(string $name)
     }
 
     /**
-     * @param string $type
+     * @param string $description
      */
-    public function setType(string $type)
+    public function setDescription(string $description)
```
Comment (Raresmldvn) [inline src/Entity/QuizTemplate.php]: same

File: src/views/admin-results-listing.phtml [hunk 3]
```diff
@@ -46,7 +63,7 @@ require_once('partials/head.phtml');
                     <?php /** @var $result UserQuizResult */ ?>
                     <th scope="row"><?php echo  $result->getQuizInstance()->getId(); ?></th>
                     <td><?php echo $result->getUser()->getName() ?> </td>
-                    <td><?php echo $result->getQuizInstance()->getQuizTemplateId() ?></td>
+                    <td><?php echo $result->getQuizInstance()->getName() ?></td>
                     <td>We don't believe that numbers represent the value of people.</td>
```
Comment (Raresmldvn) [inline src/views/admin-results-listing.phtml:67]: you should remove this eventually :))

Comment (andreiuta97) [reply src/views/admin-results-listing.phtml:67]: I've changed it in "Add score" PR :))) @Raresmldvn

**Pull request merged**